packages/example-app-base/src/components/App/App.tsx

```diff
@@ -27,7 +27,9 @@ export class AppBase extends React.Component<IAppProps, IAppState> {
     const { customizations } = appDefinition;
     const { isMenuVisible } = this.state;
 
-    const classNames = (this._classNames = getClassNames(styles, { responsiveMode, theme }));
+    const showOnlyExamples = location && location.hash.indexOf('docsExample=true') > -1;
```

Comment:
To account for SSR you have to check for `location` using `typeof window !== 'undefined' && window.location` (or preferably use the `getWindow()` utility).

Reply:
👍 will try and do that.